synapse/models/inet.py

```diff
@@ -1283,10 +1406,13 @@ def getModelDefs(self):
                         ('time', ('time', {}), {
                             'doc': 'The date and time the account performed the action.'
                         }),
-                        ('ipv4', ('inet:ipv4', {}), {
+                        ('client', ('inet:client', {}), {
```

Comment:
@invisig0th - do we want the rest of the subs for these? or just the ipv4/ipv6?

Reply:
I figured that I would add the subs for all of the props that were already on the form, but not add any others

Reply:
it'll depend on the use case, but for most of them, yea we want an inet:server / inet:client subs declared so we can use ipv4 directly

Reply:
OK, I'll leave them all as they were then.